src/main/java/com/sogeti/SoSenseiPsyche/model/Code.java

```diff
@@ -0,0 +1,34 @@
+package com.sogeti.SoSenseiPsyche.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Code {
+    List<Colour> code = new ArrayList<>();
+
+    Colour createRandomColour() {
```

Comment:
make this private. Also is it possible to move this to the Colour class? Then it will be a static method